modules/lvms-using-raid-arrays.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="lvms-using-raid-arrays_{context}"]
+= Integrating software RAID arrays with {lvms}
+
+Logical Volume Manager (LVM) does not support creating a software RAID. However, you can create the RAID array using the `mdadm` utility, and integrate the RAID array with {lvms}.
+You can integrate the RAID array with {lvms} while creating the `LVMCluster` custom resource (CR). 
+
+.Prerequisites
+
+* You created a software RAID during the {product-title} installation.
++
+[NOTE]
+====
+You can only create the RAID array during the installation of {product-title}.
+====
+
+* You have installed {lvms}.
+
+.Procedure
+
+. Open the `LVMCluster` custom resource (CR) YAML file.
++
+[NOTE]
+====
+If you have created the LVM cluster, then you can edit the existing `LVMCluster` CR.
+====
+
+. Add the path to the RAID array in the `deviceSelector` field in `LVMCluster` CR.
++
+.Example of RAID array paths in deviceSelector
+[source,yaml]
+----
+apiVersion: lvm.topolvm.io/v1alpha1
+kind: LVMCluster
+metadata:
+  name: my-lvmcluster
+spec:
+  storage:
+    deviceClasses:
+# ...
+      deviceSelector:  <1>
+        paths:
+        - /dev/md/md-var  <2>
+        - /dev/md0  <3>
+        optionalPaths:
+# ...
+----
+<1> Contains the paths to the devices used to create the LVM volume group. You can specify the device paths in `paths` field, `optionalPaths` field, or both. You cannot remove or replace a device after adding it to the LVM cluster.
+<2> Example of a path to the RAID array created using `mdadm` utility. In this example, `md-var` is the RAID array.
+<3> In this device path example, `md0` is the RAID array
+
+. Save the `LVMCluster` CR YAML file.
+
+[NOTE]
+====
+If you do not add the path to the RAID array in the `deviceSelector` field, then {lvms} does not recognize the RAID array created using the `mdadm` utility.
```

Comment:
```suggestion
If you do not add the path to the RAID array in the `deviceSelector` field, the {lvms} does not recognize the RAID array.
```

I dropped a clause as it repeats previous information.

Reply:
Updated as per the suggestion.